ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -46,6 +46,11 @@ releases:
       architecture: multi
       channel: fast
       version: "4.16"
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks